src/main/java/trello/ui/pages/BoardPage.java

```diff
@@ -37,11 +39,24 @@
     @FindBy(className = "primary")
     private WebElement addNewListButton;
 
-    @FindBy(className = "js-close-list")
+    @FindBy(className = ARCHIVE_LIST_CLASS)
     private WebElement archiveListButton;
 
-    private static final String BOARD_TITLE_XPATH = "//h2[contains(text(), \"%s\")]";
-    private static final String BOARD_MENU_SUFFIX = "/following-sibling::div";
+    @FindBy(className = "js-move-cards")
+    private WebElement moveAllCardsButton;
+
+    @FindBy(className = "js-sort-cards")
+    private WebElement sortCardsButton;
+
+    @FindBy(className = SORT_BY_NAME_CLASS)
+    private WebElement sortByCardNameButton;
+
+    private static final String ARCHIVE_LIST_CLASS = "js-close-list";
+    private static final String SORT_BY_NAME_CLASS = "js-sort-by-card-name";
+    private static final String LIST_TITLE_XPATH = "//h2[contains(text(), '%s')]";
+    private static final String LIST_MENU_SUFFIX = "/following-sibling::div";
```

Comment:
Please add the character "/" in it string.

Reply:
There is no need no add / because this constant represents only the suffix to be added to de first part of the Xpath locator.